app/lib/utils/secret_decoder.rb

```diff
@@ -0,0 +1,20 @@
+module Utils
+  class SecretDecoder
```

Comment:
I wonder if this classname is a little broad for what it does (it only works on JWT) -- maybe just `jwt_secret_decoder`? Or both decoders in a `jwt` module?

Reply:
moved both to jwt module